portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -98,6 +105,20 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     response.sendRedirect("/comments.html");
   }
 
+  /*
+   * Gets the sentiment score for a comment by using
+   * the Google Natural Language AP.
+   * @return sentiment score for comment.
+   */
+  private double getSentimentScore(String comment) throws IOException {
+    LanguageServiceClient languageService = LanguageServiceClient.create();
+    Document doc = Document.newBuilder().setContent(comment).setType(Document.Type.PLAIN_TEXT).build();
+    Sentiment sentiment = languageService.analyzeSentiment(doc).getDocumentSentiment();
+    double score = sentiment.getScore();
+    languageService.close();
+    return score;
```

Comment:
I would just inline this variable: return sentiment.getScore();

Reply:
Oh that's a really good point, I sort of created this variable for no reason here. I'll make that change right now. Thanks for pointing it out!